src/fsharp/vs/Symbols.fs

```diff
@@ -1397,7 +1397,29 @@ and FSharpMemberOrFunctionOrValue(cenv, d:FSharpMemberOrValData, item) =
 
         | V v -> 
         match v.ValReprInfo with 
-        | None -> failwith "not a module let binding or member"
+        | None ->
+            let _, tau = v.TypeScheme
+            if isFunTy cenv.g tau then
+                let typeArguments, _typ = stripFunTy cenv.g tau
+                [ for typ in typeArguments do
+                    let allArguments =
+                        if isTupleTy cenv.g typ
+                        then tryDestTupleTy cenv.g typ
+                        else [typ]
+                    yield
+                      [for typ in allArguments do
+                          match typ with
+                          | TType_var tp ->
```

Comment:
I'm not sure what this is doing. It seems to be saying "if the argument's type is a generic type parameter T, then look at the attributes of the generic type parameter to work out if the argument is optional etc."  But that doesn't make sense.

I think isOptionalArg, isOutArg and isParamArrayArg should all just be false and you shouldn't be doing this match at all.